.travis.yml

```diff
@@ -3,5 +3,4 @@ sudo: false
 rvm:
   - 2.2
   - 2.1
-  - 1.9.3
```

Comment:
@thommay are you okay with removing 1.9.3 testing?  We would like to use ruby 2.x features.